reactor-core/src/main/java/reactor/util/Loggers.java

```diff
@@ -454,6 +455,24 @@ final String format(@Nullable String from, @Nullable Object... arguments){
 			}
 			return null;
 		}
+
+		private void logWithOptionalThrowable(Level level, String format, Object... arguments) {
+			if(isLastElementThrowable(arguments)) {
+				int lastIndex = arguments.length - 1;
+				Object[] args = Arrays.copyOfRange(arguments, 0, lastIndex);
+				Throwable t = (Throwable) arguments[lastIndex];
+
+				logger.log(level, format(format, args), t);
+				return;
+			}
+
+			logger.log(level, format(format, arguments));
+		}
```

Comment:
In `JdkLogger`, if the last arguments are `Throwable` type, stacktrace is also logged.

Reply:
I'd love to see a test for this.